### PR TITLE
Release v1.2.11

### PR DESCRIPTION
**Pull request by bryanculver**
Request to merge `develop` into `main` for release v1.2.11.

# What's Changed

https://github.com/nautobot/nautobot/compare/v1.2.10...develop

# TODO

- [x] Bump version in pyproject.toml
- [x] Update date in release notes

### CODE REVIEW

File: nautobot/docs/requirements.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
-mkdocs==1.2.3
+Jinja2==3.0.3
+mkdocs==1.3.0
```
Comment (glennmatthews) [inline nautobot/docs/requirements.txt:2]: A bit of a collision here between fixes I think - it looks like while Jinja2 3.1.0 is not compatible with mkdocs 1.2.3, mkdocs 1.2.4 and/or 1.3.0 *are* compatible with Jinja2 3.1.0, so the Jinja2 pin may not be strictly necessary here.

Also, we should be pinning Jinja2 to the same version in `docs/requirements.txt` as we are pinning in `pyproject.toml`/`poetry.lock` - so in `develop` this should really be `Jinja2==2.11.3` and we should pin `Jinja2==3.0.3` in `next` until we update `next` to Jinja2 3.1.x.

File: pyproject.toml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 name = "nautobot"
 # Primary package version gets set here. This is used for publishing, and once
 # installed, `nautobot.__version__` will have this version number.
-version = "1.2.10"
+version = "1.2.11-beta.1"
```
Comment (glennmatthews) [inline pyproject.toml:5]: Change to `1.2.11` before merging. :-)

File: nautobot/docs/release-notes/version-1.2.md [hunk 3]
```diff
@@ -140,6 +140,28 @@ All models that have `slug` fields now use `AutoSlugField` from the `django-exte
 
 Just as with the UI, the `slug` can still always be explicitly set if desired.
 
+## v1.2.11 (2022-MM-DD)
```
Comment (glennmatthews) [inline nautobot/docs/release-notes/version-1.2.md:143]:
```suggestion
## v1.2.11 (2022-04-04)
```

**Review by glennmatthews — APPROVED** [review]

File: nautobot/docs/requirements.txt (hunk 1 above)
Comment (bryanculver) [inline nautobot/docs/requirements.txt:2]:
```suggestion
Jinja2==2.11.3
mkdocs==1.3.0
```

Is this your suggestion @glennmatthews?

Comment (glennmatthews) [reply nautobot/docs/requirements.txt:2]: Yes, for `develop` at this time.

**Pull request merged**